.github/workflows/ci.yml

```diff
@@ -4,7 +4,7 @@ on: [ pull_request ]
 jobs:
   tests:
     name: PHPUnit PHP ${{ matrix.php }} ${{ matrix.dependency }} (Symfony ${{ matrix.symfony }})
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-22.04
```

Comment:
What's the reasoning for not just using 'latest'? :)

Reply:
In a rare case, when GitHub decides maybe next year to switch the latest tag to ubuntu-24.04 (next LTS version), the used actions may fail / are incompatible with that new OS version.